rasa_core/policies/sklearn_policy.py

```diff
@@ -188,7 +195,13 @@ def load(cls, path: Text) -> Policy:
             ("Loaded featurizer of type {}, should be "
              "MaxHistoryTrackerFeaturizer.".format(type(featurizer).__name__))
 
-        policy = cls(featurizer=featurizer)
+        meta_file = os.path.join(path, "sklearn_policy.json")
+        if os.path.isfile(meta_file):
```

Comment:
why should this file not exist?

Reply:
Ah, I didn't notice that sklearn_policy's `model` doesn't default to `None` like keras_policy's does, so it'll always persist something. Unless you specifically instantiate your sklearn_policy with `model=None`, overriding the default model (I admit there is a super slim chance of someone doing this)

Isn't this super slim chance the only possible way to hit the warning in `sklearn_policy.persist()` though?

```
    def persist(self, path: Text) -> None:

        if self.model:
            self.featurizer.persist(path)

            filename = os.path.join(path, 'sklearn_model.pkl')
            with open(filename, 'wb') as f:
                pickle.dump(self._state, f)
        else:
            warnings.warn("Persist called without a trained model present. "
                          "Nothing to persist then!")
```